src/sandstorm/run-bundle.c++

```diff
@@ -921,8 +921,15 @@ private:
     return kj::heapString(exeName.slice(0, KJ_ASSERT_NONNULL(exeName.findLast('/'))));
   }
 
-  void changeToInstallDir() {
+  void changeToInstallDirAndCheckAccess() {
     KJ_SYSCALL(chdir(getInstallDir().cStr()));
+    if (access("../var/sandstorm", W_OK) == -1) {
+      if (errno == EACCES) {
+        KJ_FAIL_REQUIRE("Sandstorm must be run as root or a member of SERVER_USER's group.");
```

Comment:
Okay, so special message for the dev command? Is there an easily-detectable way of finding out whether Sandstorm was installed as root? Or should the message just include the full conditional?